setup.py

```diff
@@ -29,7 +29,7 @@ def read(*paths):
 
 setup(
     name='python-twitter',
-    version='2.3',
+    version='2.3.1',
```

Comment:
This is just my idea for the version jump... not sure if you use this sort of versioning but it maybe sane for such a small jump like this